server/src/Player.ts

```diff
@@ -0,0 +1,32 @@
+import Position from "./Position";
+import Rotation from './Rotation';
+
+const distanceLimit = 5;
+const zLimit = 0.5;
+
+export default class Player {
```

Comment:
as I've said, remove the validations for now

Reply:
Done.